tornjak-backend/cmd/agent/agent.go

```diff
@@ -186,8 +186,8 @@ func GetServerInfo(config *run.Config) (agentapi.TornjakServerInfo, error) {
 
 func getSocketPath(config *run.Config) string {
 	socketPath := config.Server.SocketPath
-	if config.Server.DeprecatedRegistrationUDSPath != "" {
-		socketPath = config.Server.DeprecatedRegistrationUDSPath
+	if config.Server.SocketPath != "" {
+		socketPath = config.Server.SocketPath
```

Comment:
this is redundant now. Since this was introduced because there was a change in the name from registrationUDSPath to socketPath. However, this means that we will no longer support certain versions that require this (so we should remove them from the build script as well).

Reply:
OK, so that means we don't support anything below SPIRE 1.1.0. I will remove them then.